core/Plugin/ViewDataTable.php

```diff
@@ -235,6 +235,23 @@ public function __construct($controllerAction, $apiMethodToRequestDataTable, $ov
             $report->configureView($this);
         }
 
+        $this->assignRelatedReportsTitle();
+
+        $this->config->show_footer_icons = (false == $this->requestConfig->idSubtable);
+
+        // the exclude low population threshold value is sometimes obtained by requesting data.
+        // to avoid issuing unnecessary requests when display properties are determined by metadata,
+        // we allow it to be a closure.
+        if (isset($this->requestConfig->filter_excludelowpop_value)
+            && $this->requestConfig->filter_excludelowpop_value instanceof \Closure
+        ) {
+            $function = $this->requestConfig->filter_excludelowpop_value;
+            $this->requestConfig->filter_excludelowpop_value = $function();
+        }
+
+        $this->overrideViewPropertiesWithParams($overrideParams);
```

Comment:
I'm pretty sure those two override methods should be executed after the event as otherwise the visualisation may overwrite settings that are supposed to be set by `overrideViewPropertiesWithParams ` etc.

I think the other ones should make sense to execute them before the event but haven't looked if there are any regressions 👍

Reply:
But that way it won't be possible to receive the final parameters withing the event. Or shouldn't something like my example above be possible

Reply:
Do we need maybe a `configure.end` event instead? I reckon that makes more sense maybe

Reply:
That would work as well. Will change the PR tmrrw